app/Http/Controllers/api/v1/UserController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\api\v1;
+
+use App\Http\Controllers\Controller;
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use JWTAuth;
+use Tymon\JWTAuth\Exceptions\JWTException;
+
+
+class UserController extends Controller
+{
+    public function authenticate(Request $request)
+    {
+        $credentials = $request->only('email', 'password');
+
+        try {
+            if (! $token = JWTAuth::attempt($credentials)) {
+                return response()->json(["result"=>0,'error' => 'invalid_credentials'], 401);
+            }
+        } catch (JWTException $e) {
+            return response()->json(["result"=>0,'error' => 'could_not_create_token'], 500);
+        }
+
+        return response()->json(["result"=>1,"token"=>$token]);
+    }
+
+    public function register(Request $request)
+    {
+            $validator = Validator::make($request->all(), [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Illuminate\Support\Facades\Validator' in method 'register'.](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5510282)